### PR TITLE
Sensor extrinsics

### PR DESCRIPTION
**Pull request by elrichgro**
# Changes
- Each project has a main sensor based on the project item type. Labels are created in the coordinate system of the main sensor.
- For image projects with point cloud data, the point cloud can be opened as a secondary view, or as an overlay on top of the image. The point cloud data is transformed in image coordinates so that it aligns with labels.
- Can choose different color schemes for point cloud data (height, distance from camera, image pixels).
- For point cloud projects with image data, the image can be opened as a side view. The image camera is transformed with sensor extrinsics to align with the point cloud labels.

# Testing
1. To test image features, create a new image project with 3d bounding box labels, with the item list `examples/multi-sensor/kitti-multimodal-image.json` and sensors `examples/multi-sensor/kitti-multimodal-image-sensors.json`.
- You can open a new viewer to the side to view the point cloud data.
- You can overlay the point cloud data on the image with the overlay toggle.
- You can change the point cloud color scheme with the color scheme toggle.
2. To test point cloud features, create a new point cloud project with 3d bounding box labels, with the item list `examples/multi-sensor/kitti-multimodal-pc.json` and sensors `examples/multi-sensor/kitti-multimodal-pc-sensors.json`.
- The image should show in a side view, aligned with the labels. 
- You can toggle the point cloud colors.

### CODE REVIEW

File: app/src/components/label_pane.tsx [hunk 1]
```diff
@@ -323,29 +327,22 @@ class LabelPane extends Component<Props> {
...
-            <ListItem dense disableGutters>
-              <div hidden={pane.hide}>{horizontalSplitButton}</div>
-            </ListItem>
-            <ListItem dense disableGutters>
-              {visibilityButton}
-            </ListItem>
-          </List>
-        )
-      }
+      const paneControl = (
```
Comment (Scalsol) [inline app/src/components/label_pane.tsx:330]: These elements will appear when it is a 2d image labeling project. We should not show them under this scenerios

File: app/src/components/point_cloud_canvas.tsx [hunk 2]
```diff
@@ -88,7 +96,7 @@ const fragmentShader = `
 
     void main() {
       float alpha = 0.5;
-      vec3 color = getHeatMapColor(worldPosition.z);
+      vec3 color = pointColor;
```
Comment (Scalsol) [inline app/src/components/point_cloud_canvas.tsx:99]: In the original point cloud labeling, the points that are inside a box would be highlighed as RED. We should keep this behavior.

File: examples/multi-sensor/kitti-multimodal-image.json [hunk 3]
```diff
@@ -0,0 +1,263 @@
+{
+    "frames": [
+        {
+            "name": "image_2/000000",
+            "url": "http://localhost:8686/items/kitti/tracking/training/image_02/0001/000000.png",
```
Comment (Scalsol) [inline examples/multi-sensor/kitti-multimodal-image.json:5]: change the url to a public url. Check examples\kitti\kitti_list.json for reference.

**Review by cwlroda — APPROVED** [review]
LGTM!

**Pull request merged**